docs/core_docs/docs/deno.json

```diff
@@ -0,0 +1,25 @@
+{
+  "imports": {
+    "langchain": "npm:/file:../../../langchain",
+    "langchain/": "npm:/file:../../../langchain/",
+    "@langchain/core": "npm:/file:../../../langchain-core",
+    "@langchain/core/": "npm:/file:../../../langchain-core/",
+    "@langchain/openai": "npm:/file:../../../libs/langchain-openai",
+    "@langchain/openai/": "npm:/file:../../../libs/langchain-openai/",
+    "@langchain/mistralai": "npm:/file:../../../libs/langchain-mistralai",
+    "@langchain/mistralai/": "npm:/file:../../../libs/langchain-mistralai/",
+    "@langchain/anthropic": "npm:/file:../../../libs/langchain-anthropic",
+    "@langchain/anthropic/": "npm:/file:../../../libs/langchain-anthropic/",
+    "@langchain/cohere": "npm:/file:../../../libs/langchain-cohere",
+    "@langchain/cohere/": "npm:/file:../../../libs/langchain-cohere/",
+    "@langchain/google-genai": "npm:/file:../../../libs/langchain-google-genai",
+    "@langchain/google-genai/": "npm:/file:../../../libs/langchain-google-genai/",
```

Comment:
This will need to be updated every time we add an integration package?

Reply:
Yes, along with any import someone wants to add in a notebook. I could write a script to read the notebook imports and update this @ build time but that doesn't handle runtime

Reply:
Yeah no worries, can treat it like a dep. Can you add docs though?